### PR TITLE
Tomm/wrap up

### PR DESCRIPTION
**Pull request by MuckT**
- Sign in route
- Updated comments

### CODE REVIEW

**Review by vbchomp — COMMENTED** [review]
I can work on the 401 for bearer.js during class tonight. 
And should we set the timeout for accounts to 30 or so minutes to ensure that it is active for presentation? Or will the 20 be good?

File: src/routes/user.js [hunk 1]
```diff
@@ -14,15 +15,28 @@ userRouter.post('/register', async (req, res, next) => {
   try {
     let userRecord = await users.create(req.body);
     const output = {
-      user: userRecord,
-      userToken: userRecord.token,
+      user: userRecord
```
Comment (fadnesscharlie) [inline src/routes/user.js:18]: Why do we not need userToken sent?

**Review by fadnesscharlie — APPROVED** [review]
I approve, just wondering why userToken is taken out

File: src/routes/user.js (hunk 1 above)
Comment (MuckT) [reply src/routes/user.js:18]: It's in the user object so no need for the duplication.

**Review by vbchomp — APPROVED** [review]
Looks good to merge.

**Pull request merged**